packages/jsdom-crawler/src/internals/jsdom-crawler.ts

```diff
@@ -0,0 +1,1023 @@
+import { addTimeoutToPromise, tryCancel } from '@apify/timeout';
+import { concatStreamToBuffer, readStreamToString } from '@apify/utilities';
+import {
+    BasicCrawler,
+    BasicCrawlerOptions,
+    BASIC_CRAWLER_TIMEOUT_BUFFER_SECS,
+} from '@crawlee/basic';
+import {
+    CrawlerExtension,
+    CrawlerHandleFailedRequestInput,
+    CrawlingContext,
+    enqueueLinks,
+    EnqueueLinksOptions,
+    mergeCookies,
+    ProxyConfiguration,
+    ProxyInfo,
+    Request,
+    RequestQueue,
+    resolveBaseUrl,
+    Session,
+    storage,
+    validators,
+} from '@crawlee/core';
+import { Awaitable, Dictionary, entries, parseContentTypeFromResponse } from '@crawlee/utils';
+import contentTypeParser, { RequestLike, ResponseLike } from 'content-type';
+import { gotScraping, OptionsInit, Method, TimeoutError, Request as GotRequest } from 'got-scraping';
+import { IncomingHttpHeaders, IncomingMessage } from 'http';
+import iconv from 'iconv-lite';
+import ow from 'ow';
+import util from 'util';
+import { JSDOM, DOMWindow } from 'jsdom';
+
+/**
+ * Default mime types, which CheerioScraper supports.
+ */
+const HTML_AND_XML_MIME_TYPES = ['text/html', 'text/xml', 'application/xhtml+xml', 'application/xml'];
+const APPLICATION_JSON_MIME_TYPE = 'application/json';
+const CHEERIO_OPTIMIZED_AUTOSCALED_POOL_OPTIONS = {
+    snapshotterOptions: {
+        eventLoopSnapshotIntervalSecs: 2,
+        maxBlockedMillis: 100,
+    },
+    systemStatusOptions: {
+        maxEventLoopOverloadedRatio: 0.7,
+    },
```

Comment:
Do these need to be updated?

Reply:
we can measure once we merge, for now i would use the same